engines/sci/graphics/font.cpp

```diff
@@ -38,7 +38,9 @@ GfxFontFromResource::GfxFontFromResource(ResourceManager *resMan, GfxScreen *scr
 
 	_resource = resMan->findResource(ResourceId(kResourceTypeFont, resourceId), true);
 	if (!_resource) {
-		error("font resource %d not found", resourceId);
+		// The Lighthouse demo does not contain any fonts
```

Comment:
In any case, this commit doesn't do what the commit message claims, and I don't agree that this particular error->warning change is 'needed'.

If the error is caused by the GfxText32 constructor, that seems to mean we look up a font before it's actually necessary. Can we ignore a failing load only for that single call?